kuma/wiki/forms.py

```diff
@@ -466,15 +467,25 @@ def akismet_parameters(self):
         for a model instance in
         ``RevisionAkismetSubmissionAdminForm.akismet_parameters`` method!
         """
-        language = self.cleaned_data.get('locale',
-                                         settings.WIKI_DEFAULT_LANGUAGE)
+        if self.instance and self.instance.pk and self.instance.document:
+            language = self.instance.document.locale
+        else:
+            language = settings.WIKI_DEFAULT_LANGUAGE
+
```

Comment:
The `akismet_locale` won't change. It is simply a helper to convert Django style locale codes to what akismet expects ("en-US" -> "en_us"). What should be unique per form is `akismet_parameters` and this is already set up and intended.

But... `RevisionForm` is used in many places and the data provided to it is different depending on where it is used (editing a document vs starting a new translation). The `akismet_parameters` will need to work in those cases. Or we need different forms. I'm going to follow the former and make `akismet_parameters` try to find the locale from multiple sources of data.